### PR TITLE
Fax REST API

### PR DESCRIPTION
**Pull request by lpradovera**

### CODE REVIEW

File: __recordings__/RestClient_1253868184/should-get-a-fax-instance_4196637369/recording.har [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        },
+        "response": {
+          "bodySize": 794,
+          "content": {
+            "mimeType": "application/json; charset=utf-8",
+            "size": 794,
+            "text": "{\"account_sid\":\"99999999-ddf1-4569-8b4a-111111111111111\",\"api_version\":\"v1\",\"date_created\":\"2019-01-04T16:28:33Z\",\"date_updated\":\"2019-01-04T16:29:20Z\",\"direction\":\"outbound\",\"from\":\"+14043287382\",\"media_url\":\"https://s3.us-east-2.amazonaws.com/signalwire-assets/faxes/20190104162834-831455c6-574e-4d8b-b6ee-2418140bf4cd.tiff\",\"media_sid\":\"aff0684c-3445-49bc-802b-3a0a488139f5\",\"num_pages\":1,\"price\":0.0105,\"price_unit\":\"USD\",\"quality\":\"fine\",\"sid\":\"831455c6-574e-4d8b-b6ee-2418140bf4cd\",\"status\":\"delivered\",\"to\":\"+14044455666\",\"duration\":41,\"links\":{\"media\":\"/api/laml/2010-04-01/Accounts/99999999-ddf1-4569-8b4a-111111111111111/Faxes/831455c6-574e-4d8b-b6ee-2418140bf4cd/Media\"},\"url\":\"/api/laml/2010-04-01/Accounts/99999999-ddf1-4569-8b4a-111111111111111/Faxes/831455c6-574e-4d8b-b6ee-2418140bf4cd\"}"
+          },
+          "cookies": [],
+          "headers": [
```
Comment (bryanrite) [inline __recordings__/RestClient_1253868184/should-get-a-fax-instance_4196637369/recording.har]: Lets clear out the headers again, some of it is sensitive info.

File: __recordings__/RestClient_1253868184/should-list-the-calls_545185108/recording.har [hunk 2]
```diff
@@ -0,0 +1,1637 @@
...
+          },
+          "cookies": [],
+          "headers": [
```
Comment (bryanrite) [inline __recordings__/RestClient_1253868184/should-list-the-calls_545185108/recording.har]: Lots of headers in this one.

File: __recordings__/RestClient_1253868184/should-list-the-faxes_1608614786/recording.har [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    },
+    "entries": [
+      {
+        "_id": "0fa11b350fc4b0e8ef389dfe6b30c5dd",
+        "_order": 0,
+        "cache": {},
+        "request": {
+          "bodySize": 0,
+          "cookies": [],
+          "headers": [
```
Comment (bryanrite) [inline __recordings__/RestClient_1253868184/should-list-the-faxes_1608614786/recording.har]: Headers

File: package-lock.json [hunk 4]
```diff
@@ -738,7 +860,7 @@
     },
     "babel-plugin-istanbul": {
       "version": "4.1.6",
-      "resolved": "https://registry.npmjs.org/babel-plugin-istanbul/-/babel-plugin-istanbul-4.1.6.tgz",
+      "resolved": "http://registry.npmjs.org/babel-plugin-istanbul/-/babel-plugin-istanbul-4.1.6.tgz",
```
Comment (bryanrite) [inline package-lock.json]: Any ideas why these are going to http rather than https?

**Comment by edolix** [comment]
@bryanrite @lpradovera i removed Polly and made integration tests with a mocked server ( https://github.com/httptoolkit/mockttp ). It's easy to use and implement.

The PR is ready, just a confirm i didn't push sensitive data)

**Comment by lpradovera** [comment]
LGTM. I have a long way to go in learning JS.

**Pull request merged**